command/agent/exec/exec_test.go

```diff
@@ -380,3 +385,164 @@ func TestExecServer_Run(t *testing.T) {
 		})
 	}
 }
+
+func TestExecServer_LogFiles(t *testing.T) {
```

Comment:
From what I can tell, this only tests the `stderr` case -- we should add a test for `stdout` too.